src/App.js

```diff
@@ -1,12 +1,18 @@
 import React from "react";
-import TodoList from "./components/TodoList";
+// import TodoList from "./components/TodoList";
```

Comment:
you can delete these commented lines if you don't need them

Reply:
They have been commented for testing purposes
will have to uncomment